test/module.mk

```diff
@@ -22,6 +22,32 @@ LIBTEST_DEP := \
 
 LIBTEST     := $(DIR)/lib$(MODNAME)$(MODULE_LIBEXT)
 
+# pv and looplibrary should not interfere ######################################
+LIBPV_SRC := \
+		$(DIR)/pv.cpp
+
+LIBPV_OBJ := \
+		$(patsubst %.cpp, %.o, $(filter %.cpp, $(LIBPV_SRC))) \
+		$(patsubst %.f, %.o, $(filter %.f, $(LIBPV_SRC)))
+
+LIBPV_DEP := \
+		$(LIBPV_OBJ:.o=.d)
+
+LIBPV     := $(DIR)/libpv$(MODULE_LIBEXT)
+
+$(LIBPV_DEP) $(LIBPV_OBJ): CPPFLAGS += $(LOOPFUNCFLAGS)
+
+ifeq ($(ENABLE_SHARED_LIBS),yes)
+$(LIBPV): $(LIBPV_OBJ)
+		@$(MSG)
+		$(Q)$(MODULE_MAKE_LIB_CMD) $@ $^ $(BOOSTTHREADLIBS) $(THREADLIBS) $(GSLLIBS) $(FLIBS) $(LOOPFUNCLIBS)
```

Comment:
I'm getting a linking error from this line, because I have also installed  `LoopTools` from my distribution.
Please change this line by applying the following patch:
~~~
diff --git a/test/module.mk b/test/module.mk
index 6aee65ebd..c591f75bc 100644
--- a/test/module.mk
+++ b/test/module.mk
@@ -40,7 +40,7 @@ $(LIBPV_DEP) $(LIBPV_OBJ): CPPFLAGS += $(LOOPFUNCFLAGS)
 ifeq ($(ENABLE_SHARED_LIBS),yes)
 $(LIBPV): $(LIBPV_OBJ)
                @$(MSG)
-               $(Q)$(MODULE_MAKE_LIB_CMD) $@ $^ $(BOOSTTHREADLIBS) $(THREADLIBS) $(GSLLIBS) $(FLIBS) $(LOOPFUNCLIBS)
+               $(Q)$(MODULE_MAKE_LIB_CMD) $@ $^ $(LOOPFUNCLIBS)
 else
 $(LIBPV): $(LIBPV_OBJ)
                @$(MSG)

~~~